webapp/test/clj/lipas/backend/handler_test.clj

```diff
@@ -120,7 +121,9 @@
 (defn prune-es! []
   (let [client   (:client search)
         mappings {(-> search :indices :sports-site :search) (:sports-sites search/mappings)
-                  (-> search :indices :analysis :diversity) diversity/mappings}]
+                  (-> search :indices :analysis :diversity) diversity/mappings
+                  (-> search :indices :lois :search) {:mappings {}}}]
```

Comment:
This looks a bit funny. `{:mappings {}}` should probably be `(:lois search/mappings)`?

Reply:
Added proper mappings.